packages/apps-config/src/ui/logos/index.ts

```diff
@@ -58,6 +58,7 @@ import nodePolkadex from './nodes/polkadex.svg';
 import nodePolkadot from './nodes/polkadot-circle.svg';
 import nodePolkadotJs from './nodes/polkadot-js.svg';
 import nodeRobonomics from './nodes/robonomics.svg';
+import nodeSgc from './nodes/Sgc.svg';
```

Comment:
```suggestion
import nodeSgc from './nodes/sgc.svg';
```

Case sensitivity.

Reply:
i see!!! Blame me for not being careful ! Thanks!